Doc/library/tempfile.rst

```diff
@@ -214,7 +214,7 @@ The module defines the following user-callable items:
    by child processes.
 
    Unlike :func:`TemporaryFile`, the user of :func:`mkstemp` is responsible
-   for deleting the temporary file when done with it.
+   for deleting the temporary file and closing the file descriptor.
```

Comment:
Two problems:
- This is very far away from the paragraph describing the return type (the last one, after all the param descriptions). I would suggest moving up the return type paragraph to right after the first sentence, since it's among the most useful and non-obvious pieces of information for users.
- The return type paragraph talks about an "OS-level handle", and this one about a "file descriptor". It may not be obvious to users that these are the same thing. The `os.open` docs also say "file descriptor", so I think we should use that term here too.